terraform-infra-approvals/civil-service.json

```diff
@@ -0,0 +1,8 @@
+{
+  "resources": [
+    {"type": "azurerm_servicebus_subscription_rule"}
+  ],
+  "module_calls": [
+    {"source":  "git@github.com:hmcts/terraform-module-servicebus-subscription?ref=require_session"}
```

Comment:
You are using master, u dont need this https://github.com/hmcts/civil-service/pull/2756/files#diff-544c5b351f16b7e0995a16994b27535fdbb08cd7abde0e0d96ecb3850909a256R3

Reply:
@adusumillipraveen so it can just be `source = "git@github.com:hmcts/terraform-module-servicebus-subscription` ?